keras_core/testing/test_case.py

```diff
@@ -405,6 +419,9 @@ def create_keras_tensors(input_shape, dtype, sparse):
             )
             for k, v in input_shape.items()
         }
+    raise ValueError(
+        f"Unsupported type for `input_shape`: {input_shape.__class__}"
```

Comment:
You can use `type(input_shape)` which will work with things that aren't instances

Reply:
Thanks! Fixed.